<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VirtualCity - Interface Graphique</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitre">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>VirtualCity - Gestion de Ville</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBoxInfos">
        <property name="title">
         <string>Informations Ville</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelNomVille">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>MaVille</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCycle">
           <property name="text">
            <string>Cycle: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelBudget">
           <property name="text">
            <string>Budget: 0 €</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPopulation">
           <property name="text">
            <string>Population: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelSatisfaction">
           <property name="text">
            <string>Satisfaction: 0%</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNbBatiments">
           <property name="text">
            <string>Bâtiments: 0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxRessources">
        <property name="title">
         <string>Ressources</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="labelEau">
           <property name="text">
            <string>Eau: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelElectricite">
           <property name="text">
            <string>Électricité: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelConsommationEau">
           <property name="text">
            <string>Consommation Eau: 0/cycle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelConsommationElec">
           <property name="text">
            <string>Consommation Élec: 0/cycle</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxActions">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAfficherEtat">
         <property name="text">
          <string>Afficher État</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnLancerCycle">
         <property name="text">
          <string>Lancer Cycle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnEvenement">
         <property name="text">
          <string>Événement Aléatoire</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnAjouterMaison">
         <property name="text">
          <string>Ajouter Maison</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnAjouterUsine">
         <property name="text">
          <string>Ajouter Usine</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnAjouterParc">
         <property name="text">
          <string>Ajouter Parc</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnSupprimerBatiment">
         <property name="text">
          <string>Supprimer Bâtiment</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnSauvegarder">
         <property name="text">
          <string>Sauvegarder</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnCharger">
         <property name="text">
          <string>Charger</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QPushButton" name="btnGenererRapport">
         <property name="text">
          <string>Générer Rapport</string>
         </property>
        </widget>
       </item>
             <item row="4" column="0">
                <widget class="QPushButton" name="btnAjouterService">
                 <property name="text">
                    <string>Ajouter Service</string>
                 </property>
                </widget>
             </item>
             <item row="4" column="1">
                <widget class="QPushButton" name="btnAfficherDetails">
                 <property name="text">
                    <string>Afficher Détails</string>
                 </property>
                </widget>
             </item>
                <item row="4" column="2">
                <widget class="QPushButton" name="btnActiverService">
                 <property name="text">
                    <string>Activer Service</string>
                 </property>
                </widget>
             </item>
             <item row="5" column="0">
                <widget class="QPushButton" name="btnAjouterHabitants">
                 <property name="text">
                    <string>Ajouter Habitants</string>
                 </property>
                </widget>
             </item>
             <item row="5" column="1">
                <widget class="QPushButton" name="btnRetirerHabitants">
                 <property name="text">
                    <string>Retirer Habitants</string>
                 </property>
                </widget>
             </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxBatiments">
      <property name="title">
       <string>Liste des Bâtiments</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidgetBatiments"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxSelection">
      <property name="title">
       <string>Détails de la Sélection</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="labelSelectedInfo">
         <property name="text">
          <string>Sélection: (aucune)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxLog">
      <property name="title">
       <string>Journal d'Événements</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextEdit" name="textEditLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
